8_viz/merge_lake_data.R

```diff
@@ -0,0 +1,44 @@
+
+merge_lake_data <- function(out_ind, temp_data_ind, lake_names_ind){
+  temp_dat <- feather::read_feather(sc_retrieve(temp_data_ind))
+  lakenames <- readRDS(sc_retrieve(lake_names_ind))
+
+  total_obs <- temp_dat %>%
+    group_by(nhd_id) %>%
+    summarize(n_obs = n())
+
+  lake_days <- temp_dat %>%
+    group_by(nhd_id, date) %>%
+    summarize(n_depths = n()) %>%
+    filter(n_depths >= 5) %>%
```

Comment:
```suggestion
   dplyr::filter(n_depths >= 5) %>%
```

Reply:
I'm going to leave these for now, as I (think) they will trigger a rebuild. I was putting `dplyr::` in front of select because there is another package in the pipeline that has a select function.